src/paths/opportunities.yaml

```
@@ -0,0 +1,81 @@
+post:
+  tags:
+    - Opportunités
+  summary: Ajouter une opportunité
+  description: Retourne l'opportunité créée avec son identiant, l'horodatage et les différentes estimations associé
```

:magic_wand: **chore** :
```suggestion
  description: Retourne l'opportunité créée avec son identifiant, l'horodatage et les différentes estimations associées
```